### DISCUSSION TITLE
How to upgrade devcard as realtime?

### DISCUSSION THREAD

**Discussion by nearfriend** [dailydotdev/daily#598] [Feature Requests]
Hi everyone
If you know about how to upgrade devcard in git as realtime, please help me!

**Comment by Mr-Destructive** [comment]
There's a nice article for [Adding Devcard to GitHub](https://daily.dev/blog/adding-the-daily-devcard-to-your-github-profile) by Chris Bongers. It should be more than enough for your requirement. Is it what you want?

**Reply by idoshamun** [reply]
Thank you @Mr-Destructive for sharing this guide. This is indeed the best solution to make sure your devcard gets updated. I will lock the conversation :)
